bot/cogs/spookyavatar.py

```diff
@@ -0,0 +1,45 @@
+from discord.ext import commands
+import discord
+import aiohttp
+from PIL import ImageOps
+from PIL import Image
+from io import BytesIO
+
+
+class SpookyAvatar:
+
+    """
+    A cog that spookifies an avatar.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def get(self, url):
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as resp:
+                return await resp.read()
+
+    @commands.command(name='savatar', aliases=['spookyavatar', 'spookify'], brief='Spookify an user\'s avatar.')
+    async def repository(self, ctx, user: discord.Member=None):
+        """
+        A command to print the user's spookified avatar.
+        """
+        if user is None:
+            user = ctx.message.author
+
+        embed = discord.Embed(colour=0xFF0000)
+        embed.title = "Is this you or am I just really paranoid?"
+        embed.set_author(name=str(user.name), icon_url=user.avatar_url)
+        resp = await self.get(user.avatar_url)
+        im = Image.open(BytesIO(resp))
```

Comment:
If you're up to it, check out the "For advanced users" section of #29, where the various modifications are pulled into separate methods and one is chosen at random when the command is invoked.

Reply:
@sco1, could you maybe explain a bit how 'abstract base class ImageAlteration with each alteration being a subclass' would work? I can't seem to wrap my head around that. Thanks

Reply:
For the purposes of this event it will be sufficient to start with some static helper method(s). This should lay the necessary foundation to get the intent of the feature request.

Reply:
Okay, I'll do that for now.